models/dave_worlds/worlds/dave_graded_seabed.world

```diff
@@ -80,5 +80,12 @@
 
     <plugin name="sc_interface" filename="libuuv_sc_ros_interface_plugin.so"/>
 
+    <gui fullscreen='0'>
+      <camera name='user_camera'>
+        <pose frame=''>14.26 -12.53 0.054 -0.0 0.28 2.36</pose> 
+        <view_controller>orbit</view_controller>
+        <projection_type>perspective</projection_type>
```

Comment:
My GUI was coming out darker than in the tutorial wiki, still trying to find out why, but the change is evident.

Reply:
I have no idea why, but all of the worlds come out darker on some computers.  There are some additional light sources in the world files that you can uncomment if desired (if they were uncommented by default, the scenes would be washed out on computers that they look good on now).

Reply:
Note from @scpeters: see if there is error in `/home/$USER/.gazebo/ogre.log`

Reply:
Note from William: possibly due to different driver versions. NVIDIA driver 460 in installation tutorial vs 510 that @quarkytale has. Maybe not a problem, more likely an issue if driver very out of date.

Reply:
If this turns out non-trivial and requires a longer thread of discussions, we should decouple it from this PR, so that we can unblock and move forward with this PR and continue that discussion in a separate ticket.

In that case, I would suggest @quarkytale opens an issue ticket documenting this, linking to the DVL tutorial that shows the lighter world, and this ticket showing the darker world, and to tag @WilliamLewww in that ticket to continue the conversation.